tools/cli/commands/create.py

```diff
@@ -63,10 +63,24 @@
 MOUNT_DIR="/mnt/disks/datalab-pd"
 MOUNT_CMD="mount -o discard,defaults ${{PERSISTENT_DISK_DEV}} ${{MOUNT_DIR}}"
 
+download_docker_image() {{
+  # Since /root/.docker is not writable on the default image,
+  # we need to set HOME to be a writable directory. This same
+  # directory is used later on by the datalab.service.
+  export OLD_HOME=$HOME
+  export HOME=/home/datalab
+  echo "Getting Docker credentials"
+  docker-credential-gcr configure-docker
+  docker-credential-gcr gcr-login
```

Comment:
This is the line that I think may be causing the issue you are seeing.

Reply:
Ok, I was getting the same issue without line 74, so I was hoping it would fix the issue, but let me double check and try again. Thanks!